.code-samples.meilisearch.yaml

```diff
@@ -599,6 +599,38 @@ search_parameter_guide_show_matches_position_1: |-
     .iter()
     .map(|r| r.matches_position.as_ref().unwrap())
     .collect();
+search_parameter_guide_matching_strategy_1: |-
+  let results: SearchResults<Movie> = client
+  .index('movies')
+  .search()
+  .with_query("big fat liar")
+  .with_matching_strategy(MatchingStrategies::LAST)
+  .execute()
+  .await
+  .unwrap();
+  
+  // Get the formatted results
+  let formatted_results: Vec<&Movie> = results
+    .hits
+    .iter()
+    .map(|r| r.formatted_result.as_ref().unwrap())
+    .collect();
```

Comment:
```suggestion
```

It's a very nice addition but potentially the user wants the whole object and not only the hits. For consistency with the other examples, lets not add this.